qemu/tests/raw_image_create_and_check.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import json
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest.qemu_io import QemuIOSystem
+from virttest.qemu_storage import QemuImg
+from virttest.utils_numeric import normalize_data_size
+
+
+def run(test, params, env):
+    """
+    qemu-img supports to create a raw image file.
+    1. Create test image, and  write 1M "1" into
+       the test image via qemu-io.
+    2. Check the image virtual and actual size info.
+    :param test: Qemu test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    def _qemu_io(img, cmd):
+        """Run qemu-io cmd to a given img."""
+        logging.info("Run qemu-io %s" % img.image_filename)
+        try:
+            QemuIOSystem(test, params, img.image_filename).cmd_output(cmd, 120)
+        except process.CmdError:
+            test.fail("qemu-io to '%s' failed." % img.image_filename)
+
+    def _check_img_size(img_info, defined_size, size_key):
+        """Check the size info of the image"""
+        logging.info("Check the '%s' size info of %s"
+                     % (size_key, source.image_filename))
+        defined_size = normalize_data_size(defined_size, "B")
+        get_size = img_info[size_key]
+        if int(defined_size) != int(get_size):
+            test.fail("Got unexpected size '%s', expected size is '%s'"
+                      % (get_size, defined_size))
```

Comment:
I thought you could use` for ... in `and `zip `to deal with the `actual-size` and `virtual-size `at the same time.

Reply:
Updated, thanks Yongxue.